crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -91,6 +93,21 @@ impl Persistence {
             .map_err(Error::DbError)
     }
 
+    /// Saves the surplus capturing jit order owners to the DB
+    pub async fn save_surplus_capturing_jit_orders_orders(
+        &self,
+        auction_id: AuctionId,
+        surplus_capturing_jit_order_owners: &[Address],
```

Comment:
let's make persistence function receive domain types and do conversion from domain to database type inside `save_surplus_capturing_jit_orders_orders`

so `surplus_capturing_jit_order_owners: &[domain::eth::Address]`